### PR TITLE
add myself as a reviewer

### PR DESCRIPTION
**Pull request by avinal**
Adding myself as a reviewer. It will be helpful for speeding up PR reviews and testing.

Signed-off-by: Avinal Kumar <avinal@redhat.com>

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/ok-to-test

I created a pr to add you to the org.

The progression goes like this: 
org member -> reviewer

**Comment by avinal** [comment]
> Where is that PR? Maybe I can help upvote it.

It got merged :)

**Review by khrm — COMMENTED** [review]
@dibyom @alan-ghelardi 
Please give lgtm so that we can have cross-org approval.

File: OWNERS [hunk 1]
```diff
@@ -18,5 +18,6 @@ approvers:
 reviewers:
 - dibyom
 - alan-ghelardi
+- avinal
```
Comment (khrm) [inline OWNERS:21]: Can you put your name on the approvers list above?

Also, add me here.  If @enarha is interested, add him also.

Comment (avinal) [reply OWNERS:21]: Ack, waiting for Emil's response.

Comment (enarha) [reply OWNERS:21]: Sure. Feel free to add me as a reviewer as well.

**Pull request merged**